conf/config.yaml

```diff
@@ -0,0 +1,77 @@
+defaults:
+  - board:
+    - arty-a100t
+    - arty-a35t
+    - basys3
+    - chandalar
+    - icebreaker
+    - nexys-video
+    - nexys
+    - quickfeather
+    - tinyfpgab2
+    - zybo
+  - project:
+    - axi-lite-reg
+    - baselitex-nexys-video
+    - baselitex
+    - blinky
+    - bram-n1
+    - bram-n2
+    - bram-n3
+    - daisho-usb3
+    - dram-test-64x1d
+    - hamsternz-displayport
+    - hamsternz-hdmi
+    - ibex
+    - iir
+    - jpeg-qnr
+    - murax
+    - oneblink
+    - picorv32
+    - picosoc-simpleuart
+    - picosoc-spimemio
+    - picosoc
+    - rsdecoder
+    - up5k-nes
+    - vexriscv-smp
+    - vexriscv
+    - zynq-ps7-counter
+  - toolchain:
+    - nextpnr-ice40
+    - nextpnr-xilinx-fasm2bels
+    - nextpnr-xilinx
+    - quicklogic
+    - vivado
+    - vpr-fasm2bels
+    - vpr
+    - yosys-vivado-uhdm
+    - yosys-vivado
```

Comment:
One of the advantages of Hydra is that it lets you replace whole config trees by overriding config groups.
By declaring such large lists here, you make it difficult to override.
One option I can think of is to leverage recursive defaults list for each of the options here.

As an  example, take a look at the internal Hydra release tool [here](https://github.com/facebookresearch/hydra/tree/master/tools/release), see how it define [sets of packages.](https://github.com/facebookresearch/hydra/tree/master/tools/release/conf/set) that can be overridden together.

Zooming out one step: the config structure seems odd to me. I know nothing about this project but it feels like you are lumping many unrelated things into the same config group. maybe group them into subgroups?